sage/pallet/src/lib.rs

```diff
@@ -313,7 +313,9 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			new_config: GeneralConfigOf<T, I>,
 		) -> DispatchResult {
-			Self::ensure_organizer(origin)?;
+			let signer = ensure_signed(origin)?;
+			Self::ensure_organizer(&signer)?;
```

Comment:
I know that we have to write two lines each time here, but in general, I believe that the code is more ergonomic if we make the `ensure_organizer` just take an account id. The benefit can be seen further below.